### PR TITLE
Expose settings data

### LINKED ISSUE #11: [VIT] recover the committee list from block0
be able to recover the committee list from the block0

depends on input-output-hk/chain-libs#329

### LINKED ISSUE #41: [VIT] expose the blockchain settings data
- [x] get the blockchain start datetime
- [x] the slot duration
- [x] num block per epoch
- [x] discriminant (and to fix in: https://github.com/input-output-hk/chain-wallet-libs/blob/aa0a690dc5f542b0b51a155c5561fe6272369481/bindings/wallet-core/src/wallet.rs#L118-L122)
- [ ]  and #11

### PR DESCRIPTION
**Pull request by danielSanchezQ**
closes #11
closes #41

### CODE REVIEW

File: bindings/wallet-core/src/wallet.rs [hunk 1]
```diff
@@ -115,11 +115,7 @@ impl Wallet {
     /// in or you may see unexpected behaviors
     ///
     pub fn convert(&mut self, settings: Settings) -> Conversion {
-        let address = self
-            .account
-            .account_id()
-            // TODO: get this from the settings
-            .address(chain_addr::Discrimination::Production);
+        let address = self.account.account_id().address(settings.discrimination());
```
Comment (NicolasDP) [inline bindings/wallet-core/src/wallet.rs:118]: nice <3

File: wallet/src/blockchain.rs [hunk 2]
```diff
@@ -33,4 +34,14 @@ impl Settings {
 
         value > minimal_value
     }
+
+    // extract the block0 date-time
+    // seconds in unix time (seconds elapsed since 1-Jan-1970)
+    pub fn start_date_time(&self) -> u64 {
+        self.static_parameters.block0_start_time.0
+    }
```
Comment (NicolasDP) [inline wallet/src/blockchain.rs:42]: convert that to `SystemTime` please.

File: wallet/src/blockchain.rs [hunk 3]
```diff
@@ -33,4 +39,18 @@ impl Settings {
...
+    // seconds in unix time (seconds elapsed since 1-Jan-1970)
+    pub fn start_date_time(&self) -> SystemTime {
+        SystemTime::UNIX_EPOCH + Duration::from_secs(self.static_parameters.block0_start_time.0)
+    }
+
+    pub fn discrimination(&self) -> Discrimination {
+        self.static_parameters.discrimination
+    }
+
+    pub fn time_era(&self) -> &TimeEra {
```
Comment (danielSanchezQ) [inline wallet/src/blockchain.rs:53]: I'm not sure if with this is enough, or should I make some methods to explicitly expose the data inside?

**Comment by NicolasDP** [comment]
can you add an accessor to get the `&[Committee]` too?

**Comment by NicolasDP** [comment]
that way you close both #41 and #11

File: wallet/src/blockchain.rs (hunk 3 above)
Comment (NicolasDP) [reply wallet/src/blockchain.rs:53]: it's more than enough for now, having them available around here allows us to know what we can do for the next steps.

**Comment by danielSanchezQ** [comment]
I was gonna make another PR for that, but I can add it here.

File: wallet/src/blockchain.rs [hunk 4]
```diff
@@ -1,14 +1,19 @@
+use chain_addr::Discrimination;
+use chain_impl_mockchain::vote::CommitteeId;
 use chain_impl_mockchain::{
     block::Block,
     fee::FeeAlgorithm as _,
     ledger::{Error, Ledger, LedgerParameters, LedgerStaticParameters},
     transaction::Input,
 };
```
Comment (NicolasDP) [inline wallet/src/blockchain.rs:8]:
```suggestion
use chain_impl_mockchain::{
    block::Block,
    fee::FeeAlgorithm as _,
    ledger::{Error, Ledger, LedgerParameters, LedgerStaticParameters},
    transaction::Input,
    vote::CommitteeId,
};
```

Comment (danielSanchezQ) [reply wallet/src/blockchain.rs:8]: Auto import, I need to have more attention to this stuff.

**Pull request merged**